plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/ScrolledPageContent.java

```diff
@@ -21,7 +21,7 @@
 import org.eclipse.ui.forms.widgets.SharedScrolledComposite;
 
 
-public class ScrolledPageContent extends SharedScrolledComposite {
+class ScrolledPageContent extends SharedScrolledComposite {
 
   public ScrolledPageContent(Composite parent) {
```

Comment:
methods/constructors don't need to be public here either

Reply:
Actually, we don't really need a separate class here.  For the future to use in other classes I made it separately. :)